explorer/templates/explorer/base.html

```diff
@@ -20,6 +20,27 @@
 </head>
 
 <body>
+{% if vite_dev_mode %}
+    <div class="vite-not-running-canary" style="text-align:center;">
```

Comment:
Is the idea here that is vite is running, the CSS for this class is available and therefore this gets hidden?

Reply:
Yes exactly! It works well and is reasonably helpful I think. See the screenshot.